.gitignore

```diff
@@ -1,11 +1,12 @@
 *.elf
 *.pdx
+*.dylib
```

Comment:
good addition! and same with the `**/` prefixes, could you re-add the trailing new-line in this file?